data/moduleStaticProps.ts

```diff
@@ -0,0 +1,50 @@
+import compareVersions from 'compare-versions'
+import {
+  extractModuleInfo,
+  getModuleMetadata,
+  getSubmissionCommitOfVersion,
+  ModuleInfo,
+} from './utils'
+
+export interface VersionInfo {
+  version: string
+  submission: {
+    hash: string
+    authorDate: string
+    authorDateRel: string
+  }
+  moduleInfo: ModuleInfo
+  isYanked: boolean
+  yankReason: string | null
+}
+
+// [module]/[version] needs to reuse the same logic
+export const getStaticPropsModulePage = async (
```

Comment:
nit: do we care about style of assigning to a function expression vs. using the `function` keyword? 
`export async function getStaticPropsModulePage() {}` ?

Reply:
Yeah, it should be consistently the `export const getStaticPropsModulePage = () => {}` form in the codebase right now.